platforms/hyperledger-indy/configuration/roles/create/helm_component/domain_genesis/tasks/main.yaml

```diff
@@ -0,0 +1,126 @@
+##############################################################################################
+# This role creates the domain genesis file for organization
+##############################################################################################
+---
+##############################################################################################
+# This tasks ensures the directory of ConfigMap existance, if not exits it creates a new one
+- name: Ensures {{ release_dir }}/{{ component_type }}/{{ component_name }} dir exists
+  vars:
+    component_name: "{{ organizationItem.name }}-domain-transactions-genesis"
+    gitops: "{{ organizationItem.gitops }}"
+    release_dir: "{{ playbook_dir }}/../../../{{ gitops.release_dir }}"
+  file:
+    path: "{{ release_dir  }}/{{ component_name }}/"
+    recurse: yes
+    state: directory
+  loop: "{{ network['organizations'] }}"
+  loop_control:
+    loop_var: organizationItem
+
+################################################################################################
+# This tasks gets data from vault
+- name: Generate domain genesis for organization
+  shell: |
+    domain_genesis=""
+    first_global_trustee_did=""
+    seqNo=1
+
+    {% if network['organizations'] is defined %}
+      {% for organization in network['organizations'] %}
+        first_trustee_in_org_did=""
+
+        {% if organization.services.trustees is defined %}
+          {% for trustee in organization.services.trustees %}
+            JSON_STRING=""
+            trustee_did=$(curl --header "X-Vault-Token:{{ ac_vault_tokens[organization] }}" "{{ organization.vault.url }}"/v1/"{{ organization.name }}"/trustees/"{{ trustee.name }}"/identity/public | jq -r 'if .errors then . else .data.did end');
```

Comment:
Shouldn't be ac_vault_tokens[organization.name] ? Same below...

Reply:
Right. Updated